joomla/joomla-cms
# [J4] Fix Favicon Appereance in Chrome

Pull Request for Issue #34767.

Sorry for the wrong commit message

### Summary of Changes
The Issue describes a problem in Chrome that the favicon svg is not displayed correctly when using darkmode.
Confirming this issue I researched at officially Chrome should meanwhile support the prefers-color-scheme media query. But altought I updated to the latest Chrome Version Version 101.0.4951.41 (Official Build) (arm64) it does not.

So I found another workaround where the specific favicon can be added by using media as attribute in the link. 


### Testing Instructions
Enable Dark mode in Chrome
Check if Joomla Default Favicon is white on dark ground in Frontend (using Cassiopeia) and Backend (using atum) and on login, error and offline page

### Actual result BEFORE applying this Pull Request
The Default Joomla Favicon shows black on black in Frontend (using Cassiopeia) and Backend (using atum)
<img width="30" alt="grafik" src="https://user-images.githubusercontent.com/828371/166147200-4ad4d2f4-6582-4fff-b004-c364c27546de.png">

### Expected result AFTER applying this Pull Request
<img width="39" alt="grafik" src="https://user-images.githubusercontent.com/828371/166147299-7e461405-4f6f-4969-8f1a-0ca62318c6f1.png">


### Documentation Changes Required
no
